app/assets/javascripts/controllers/toolbar_controller.js

```diff
@@ -44,9 +44,7 @@
   */
   function initEndpoints(MiQEndpointsService) {
     var urlPrefix = '/' + location.pathname.split('/')[1];
-    if (urlPrefix) {
-      MiQEndpointsService.rootPoint = urlPrefix;
-    }
+    MiQEndpointsService.rootPoint = urlPrefix;
```

Comment:
This looks like a bug, maybe the test should have been on `location.pathname.split('/')[1]`?